### DISCUSSION TITLE
Thanks so much for writing this library!

### DISCUSSION THREAD

**Discussion by pirate** [MrThearMan/django-signal-webhooks#90] [General]
Just added it to [ArchiveBox](https://github.com/ArchiveBox/ArchiveBox/), only took me 5min to get it running and it worked on the first try.

It's exactly what we needed, thanks!

![image](https://github.com/MrThearMan/django-signal-webhooks/assets/511499/00efed19-41fe-470e-b933-6c6e3eb01138)

![image](https://github.com/MrThearMan/django-signal-webhooks/assets/511499/ae92e4f1-5fbc-456c-a66b-caeec18442e8)

**Comment by matti-lamppu** [comment]
Thanks, glad you liked it! 😀

**Comment by pirate** [comment]
I just found your others as well, they're all awesome too 🚀🚀🚀:

Every one of these perfectly solves a real problem I happen to have had recently, the timing is incredible.

- ⭐️ https://mrthearman.github.io/django-admin-data-views/setup/ ➡️ just implemented this as well: https://github.com/ArchiveBox/ArchiveBox/pull/1420
- https://github.com/MrThearMan/django-settings-holder
- https://mrthearman.github.io/django-post-fetch-hook/usage/
- https://github.com/MrThearMan/django-subforms
- https://github.com/MrThearMan/django-lookup-property
- https://github.com/MrThearMan/sqlite3-cache
